acceptance_tests/features/steps/enrolment_status_disable.py

```diff
@@ -4,28 +4,30 @@
 
 
 @given('the internal user is on the ru details page')
+@when(u'the internal user is back on the ru details page')
```

Comment:
Why is this a `u` string?

Reply:
when you miss out an entry an `behave`  then it prompts you to add it, and shows a `u` based string, which in this case isn't needed I guess.

Reply:
Have changed it accordingly